packages/stryker-elements/src/exports/lib/atoms/copy-text.ts

```diff
@@ -0,0 +1,10 @@
+import { defineElement } from '../../../define-element';
+import { CopyText } from '../../../lib/atoms/copy-text';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sme-copy-text': CopyText;
+  }
+}
+
+defineElement('sme-copy-text', CopyText);
```

Comment:
I think, during the hacktoberfest, the idea was that you could individually import the components and customize them. I'm not sure if this is needed anymore or if it's something that we want. It works quite cumbersome at the moment so I'm leaning towards removing them.

Reply:
Importing individual components is a good feature. But I don't see how this works towards it. The component by itself is not much use as you'd have to call `defineElement` anyway, but with a different name

Reply:
If you are interested: https://atomicdesign.bradfrost.com/chapter-2/

Reply:
> If you are interested: https://atomicdesign.bradfrost.com/chapter-2/

This is very enlightening! Definitely add this to the contributing.md

Reply:
> Importing individual components is a good feature. But I don't see how this works towards it. The component by itself is not much use as you'd have to call defineElement anyway, but with a different name

Hmm then I'm not sure, I think it could go honestly. I'd rather work with the decorator.